rs/config/src/subnet_config.rs

```diff
@@ -161,6 +161,10 @@ const DEFAULT_RESERVED_BALANCE_LIMIT: Cycles = Cycles::new(5 * T);
 /// 1/10th of a round.
 pub const DEFAULT_UPLOAD_CHUNK_INSTRUCTIONS: NumInstructions = NumInstructions::new(200_000_000);
 
+/// Baseline cost for creating or loading a canister snapshot.
+pub const DEFAULT_CANISTERS_SNAPSHOT_BASELINE_INSTRUCTIONS: NumInstructions =
```

Comment:
Could split into two fees, one for take canister snapshots another one for load canister snapshots.  
The value was chosen arbitrarily.